cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -141,7 +140,9 @@ def finish_all_flow_cells_temp(self) -> bool:
 
     def store_flow_cell_data(self, parsed_flow_cell: FlowCellDirectoryData) -> None:
         """Store data from the flow cell directory in status db and housekeeper."""
-        store_flow_cell_data_in_status_db(parsed_flow_cell=parsed_flow_cell, store=self.status_db)
+        store_flow_cell_data_in_status_db(
```

Comment:
I don't like involving housekeeper in a function related to status_db. Maybe you could fetch the samplesheet early in the finish command set it to an attribute of parsed_flow_cell?

Reply:
fixed